test/powershell/engine/Module/Module.Basic.Tests.ps1

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Module basic tests" -tags "CI" {
```

Comment:
Maybe put the _single_ test in `test\powershell\Modules\Microsoft.PowerShell.Core\Import-Module.Tests.ps1`?